src/TerminalFighter/MainMenu/MainMenu.cpp

```diff
@@ -6,17 +6,62 @@
 MainMenu::MainMenu(std::string font_path, int width, int height)
     : title_(
         "Terminal Fighter",
-        {255, 255, 255},
-        width / 4,
-        height / 5,
-        width / 2,
-        height / 4,
+        WHITE,
+        0.25 * width,
+        0.2 * height,
+        0.5 * width,
+        0.25 * height,
         font_path,
-        68) {
+        68)
+    , start_(
+        "Start",
+        WHITE,
+        0.45 * width,
+        0.6 * height,
+        0.1 * width,
+        0.1 * height,
+        font_path,
+        68)
+    , quit_(
+        "Quit",
+        GRAY,
+        0.45 * width,
+        0.7 * height ,
+        0.1 * width,
+        0.1 * height,
```

Comment:
Comments explaining the reasoning for these numbers would be helpful. It's not immediately clear what they do.